util/Android/textviewsizecalculator.ts

```diff
@@ -0,0 +1,35 @@
+import AndroidConverter from './unitconverter';
+import AndroidConfig from './androidconfig';
+import TypeValue from './typevalue';
+
+const NativeTextView = requireClass('android.widget.TextView');
+const NativeView = requireClass('android.view.View');
+
+namespace SizeCalculator {
```

Comment:
unnecessary namespace and `textviewsizecalculator` function and filename should be same

Reply:
Done, exported as normal function and changed index.ts